tensorflow/contrib/distribute/python/parameter_server_strategy_test.py

```diff
@@ -178,6 +179,77 @@ def model_fn():
         self.assertEqual(z_val, 43.0)
         self.assertEqual(f_val, 46.0)
 
+  def _test_device_assignment_distributed_enable_partitioner(self,
+                                                             task_type,
+                                                             task_id,
+                                                             num_gpus):
+    d, _, sess_config = self._get_test_objects(task_type, task_id, num_gpus)
+    num_shards = len(d.parameter_devices)
+    partitioner = partitioned_variables.fixed_size_partitioner(num_shards)
+    with ops.Graph().as_default(), \
+         self.cached_session(target=self._default_target,
+                             config=sess_config) as sess, \
+         d.scope():
+
+      # Define a variable outside the call_for_each_tower scope. This is not
```

Comment:
It is fine to create the variable as long as it is under distribution strategy's scope. Could you remove this comment?